### DISCUSSION TITLE
GitHub sponsors and billing information

### DISCUSSION THREAD

**Discussion by chkpnt** [community/community#61151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In terms of privacy: Is the billing information (real name, address, credit card number) shared with the sponsored person/organization or not?

(the answer to this Q is only about the visibility in the GitHub UI: https://github.com/orgs/community/discussions/57067)

**Comment by anjuan** [comment, marked as answer]
Hi @chkpnt. No, we do not share the real name, address, or credit card number with the sponsored person/organization. However, if your GitHub handle is the same as your real name or you have your real name on your GitHub profile, then that information will be shared with the sponsored person/organization.
